src/background.ts

```diff
@@ -118,6 +120,13 @@ app.on('activate', () => {
 })
 
 const startUpTorOnPort = (port: number) => {
+  fs.chmodSync(path.join(__dirname, 'bin', 'Tor', 'libcrypto.so.1.1'), 0o755)
+  fs.chmodSync(path.join(__dirname, 'bin', 'Tor', 'libevent-2.1.6.dylib'), 0o755)
+  fs.chmodSync(path.join(__dirname, 'bin', 'Tor', 'libevent-2.1.so.6'), 0o755)
+  fs.chmodSync(path.join(__dirname, 'bin', 'Tor', 'libssl.so.1.1'), 0o755)
+  fs.chmodSync(path.join(__dirname, 'bin', 'Tor', 'tor'), 0o755)
+  fs.chmodSync(path.join(__dirname, 'bin', 'Tor', 'tor.real'), 0o755)
```

Comment:
missing the `dll's` & `.exe`

Reply:
Correct will have to clean up anyway cause this is ugly af

Reply:
Theres no need to do the exe & dlls cause its not a windows thing apparently 👯‍♂️